src/Compilers/Core/Portable/InternalUtilities/EnumerableExtensions.cs

```diff
@@ -350,7 +350,7 @@ public static bool Contains<T>(this IEnumerable<T> sequence, Func<T, bool> predi
             return sequence.Any(predicate);
         }
 
-        public static bool Contains(this IEnumerable<string> sequence, string s)
+        public static bool Contains(this IEnumerable<string?> sequence, string? s)
```

Comment:
>Contains [](start = 27, length = 8)

Can we remove this method and use `Enumerable.Contains<T>()` instead?

Reply:
Semantic wise they're basically identical but that was true from the start of Roslyn. My suspicion is that this was added because it has better peformance characteristics and it's a hot path for us. Specifically it avoid virtual calls where `Enumerable.Contains` is not guaranteed to do so.